ytypes/leafref_test.go

```diff
@@ -152,7 +152,7 @@ func TestValidateLeafRefData(t *testing.T) {
 						Kind: yang.LeafEntry,
 						Type: &yang.YangType{
 							Kind: yang.Yleafref,
-							Path: "../../../leaf-list",
```

Comment:
Was this test case buggy? I guess it was the wrong error message, but this never actually did a valid lookup.